### PR TITLE
add multilabel_probabilities

### PR DESCRIPTION
**Pull request by KimBue**
Signed-off-by: Kim Buergl

### CODE REVIEW

File: small_text/classifiers/classification.py [hunk 1]
```diff
@@ -154,7 +160,7 @@ def __init__(self, linearsvc_kwargs=None):
         self.linearsvc_kwargs = dict() if linearsvc_kwargs is None else linearsvc_kwargs
         super().__init__(**self.linearsvc_kwargs)
 
-    def predict(self, x, return_proba=False):
+    def predict(self, x, return_proba=False, **kwargs):
```
Comment (chschroeder) [inline small_text/classifiers/classification.py:163]: I don't understand this one? Is this a remnant from a previous change?

**Review by chschroeder — CHANGES_REQUESTED** [review]
Thanks, for your effort!

This is going into the right direction. I added some suggestions and found one place where I think some logic is missing.

Otherwise all the classifiers must be adapted (and tested). I would suggest the use the kwarg of the name `sparse_proba` here (with a default of True).

File: small_text/utils/classification.py [hunk 2]
```diff
@@ -47,7 +47,7 @@ def get_splits(train_set, validation_set, multi_label=False, validation_set_size
     return sub_train, sub_valid
 
 
-def prediction_result(proba, multi_label, num_classes, enc=None, return_proba=False):
+def prediction_result(proba, multi_label, num_classes, enc=None, return_proba=False, multilabel_probabilities='thresholded'):
```
[the remark this answers is not in the document]
Comment (KimBue) [reply small_text/utils/classification.py]: done

File: small_text/classifiers/classification.py (hunk 1 above)
Comment (KimBue) [reply small_text/classifiers/classification.py]: yep, I removed it now

**Comment by chschroeder** [comment]
Closing this after consultation with @KimBue. This will be continued but at another time.

**Pull request closed without merging**